staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -4150,7 +4150,7 @@ type ServiceSpec struct {
 	// externalName is the external reference that discovery mechanisms will
 	// return as an alias for this service (e.g. a DNS CNAME record). No
 	// proxying will be involved.  Must be a lowercase RFC-1123 hostname
-	// (https://tools.ietf.org/html/rfc1123) and requires Type to be
+	// (https://tools.ietf.org/html/rfc1123) and requires Type to be ExternalName
```

Comment:
```suggestion
	// (https://tools.ietf.org/html/rfc1123) and requires Type to be "ExternalName".
```

Reply:
I will make the suggested change and regenerate the other files. 

Is there a convention doc I can reference that explains how to annotate these comments? From the suggestion, it appears as though field names should be surrounded by backticks and field values are surrounded by double quotes. I noticed that this convention isn't followed for many other field descriptions.

For example, should similar changes be made in the comment for the `HealthCheckNodePort` field in ServiceSpec?

Reply:
The rule of thumb is that the comments are used to generate user-facing documents. Users will see field names encoded in the JSON (OpenAPI spec), not the Golang field name. That is the reason we need to change "Type" to `type`. The ExternalName is a string value to be assigned to the `type` field, adding the double quotes is for clarification.